v0/destinations/google_adwords_remarketing_lists/transform.js

```diff
@@ -20,6 +21,24 @@ const {
   TYPEOFLIST
 } = require("./config");
 
+const { MappedToDestinationKey } = require("../../../constants");
+
+const getSchemaForEventMappedToDest = message => {
```

Comment:
we have the same piece of code for `fb_custom_audience`. Can it be carved out and moved to util and be used anywhere else.

Reply:
Let's mention the new method used for VDM in the notion task or a separate doc so we don't add any redundant code.